betelgeuse.py

```diff
@@ -20,11 +20,17 @@
 import ssl
 import testimony
 import time
+import sys
 
 from collections import Counter
-from pylarion.exceptions import PylarionLibException
-from pylarion.work_item import TestCase, Requirement
-from pylarion.test_run import TestRun
+try:
+    from pylarion.exceptions import PylarionLibException
+    from pylarion.work_item import TestCase, Requirement
+    from pylarion.test_run import TestRun
+except ImportError as exception:
```

Comment:
@elyezer agree, 

I want to convey user that if Pylarion and its dependencies are not installed then install them. 

Should I check individual modules then and notify users ?